### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: zingocli/regtest/bin/.gitinclude [hunk 1]
```diff
@@ -0,0 +1 @@
+f
```
Comment (AloeareV) [inline zingocli/regtest/bin/.gitinclude:1]: What does this mean?

**Review by AloeareV — APPROVED** [review]
This is great! One question:

**Review by fluidvanadium — DISMISSED** [review]

File: zingocli/regtest/bin/.gitinclude (hunk 1 above)
Comment (zancas) [reply zingocli/regtest/bin/.gitinclude:1]: Include that directory.

**Review by fluidvanadium — APPROVED** [review]

**Comment by fluidvanadium** [comment]
clippy

**Pull request merged**